### PR TITLE
release 0.0.1-rc.23-v1

### PR DESCRIPTION
**Pull request by ben-kaufman**

### CODE REVIEW

File: migration.json [hunk 1]
```diff
@@ -132,55 +132,55 @@
     },
     "dao": {
       "0.0.1-rc.16": {
-        "name": "Concomitant Opera singer",
+        "name": "Obtuse Sawfiler",
```
Comment (jellegerbrandy) [inline migration.json]: Why do all these names change all the time?

Comment (ben-kaufman) [reply migration.json]: Because we made them random

Comment (jellegerbrandy) [reply migration.json]: But you are regenerating them each time?

Comment (jellegerbrandy) [reply migration.json]: apparently yes

Comment (ben-kaufman) [reply migration.json]: yes

**Pull request merged**